src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -240,6 +241,12 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->registerAliasForArgument('parameter_bag', PsrContainerInterface::class);
 
+        $loader->load('process.php');
+
+        if (!class_exists(RunProcessMessageHandler::class)) {
```

Comment:
Maybe put load into the if statement and remove negation? Why load if we know there is no such class?

Reply:
I followed the standard of the rest of this file.